kernel/packages/decentraland-ecs/src/decentraland/Types.ts

```diff
@@ -420,6 +420,7 @@ export type ProfileForRenderer = {
   name: string
   description: string
   email: string
+  ethAddress: string
```

Comment:
This is SDK. I don't think we need to touch this for this use case.

Reply:
Just tested without that and it works, removed.